README.md

```diff
@@ -118,24 +118,32 @@ Check out the [Quickstart](https://huggingface.co/docs/diffusers/quicktour) to l
-- See [Good first issues](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) for general opportunities to contribute
-- See [New model/pipeline](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+pipeline%2Fmodel%22) to contribute exciting new diffusion models / diffusion pipelines
-- See [New scheduler](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+scheduler%22)
+You can look out for [issues](https://github.com/ModelsLab/diffusers_plus_plus/issues) you'd like to tackle to contribute to the library.
 
-Also, say 👋 in our public Discord channel <a href="https://discord.gg/G7tWnz98XR"><img alt="Join us on Discord" src="https://img.shields.io/discord/823813159592001537?color=5865F2&logo=discord&logoColor=white"></a>. We discuss the hottest trends about diffusion models, help each other with contributions, personal projects or just hang out ☕.
+- See [Good first issues](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) for general opportunities to contribute.
+- See [New model/pipeline](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+pipeline%2Fmodel%22) to contribute exciting new diffusion models/diffusion pipelines.
+- See [New scheduler](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+scheduler%22).
+- See [Bug](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug%22) if something isn't working.
+- See [Documentation](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22documentation%22) for improvements or additions to documentation.
+- See [Duplicate](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22duplicate%22) if this issue or pull request already exists.
+- See [Enhancement](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22enhancement%22) for new feature or request.
+- See [Help wanted](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) if extra attention is needed.
+- See [Invalid](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22invalid%22) if something doesn't seem right.
+- See [Question](https://github.com/ModelsLab/diffusers_plus_plus/issues?q=is%3Aopen+is%3Aissue+label%3A%22question%22) if further information is requested.
 
+Also, say 👋 in our public Discord channel <a href="https://discord.gg/HPN4TVHR"><img alt="Join us on Discord" src="https://img.shields.io/discord/823813159592001537?color=5865F2&logo=discord&logoColor=white"></a>. We discuss the hottest trends about diffusion models, help each other with contributions, personal projects or just chill out 🔥.
 
 ## Popular Tasks & Pipelines
```

Comment:
We can add plus pipelines here for easy access, maybe removing the others

Reply:
added